q2_feature_table/_conditional_filter.py

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+import biom
+
+
+def filter_features_conditionally(table: biom.Table,
+                                  prevelance: float, abundance: float
+                                  ) -> biom.Table:
+    """
+    A function to perform joint filtering because it makes life better
+    """
+    num_observations, num_samples = table.shape
+    prevelance = prevelance * num_samples
+
+    # Calculates the filteering parameters on the original table
+    def _filter_f(v, id_, md):
+        return (v > abundance).sum() >= prevelance
+
+    # Normalized the table to get the prevelance
+    table_n = table.copy().norm(axis='sample', inplace=False)
```

Comment:
Since you are not normalizing the table in place, I believe the call to `.copy()` can be dropped.

Reply:
Biom normalization is tenacious. I run this function probably 3 times a month (I think the PR is copied directly from a gist I've got on my computer but not github). Through trial and error, I've found that if you copy before you normalize you insure your original table remains the same. If you don't, there's a non-zero probability biom will try to be friendly and helpful normalize your data for you anyway because you must have been wrong. `table.copy().norm(inplace=False)` makes sure that you keep your original table intact, which is one of the key pieces to this function. 

I can add some more documentation if that would help clear up the confusion, but the `.copy()` here is intentional and crucial.

Reply:
That's interesting!  Thanks for clarifying. I wonder why that is.  Nothing in the BIOM code jumps out to me. After a cursory search, the only thing I found where I could see this happening is if you made a call to `Table.collapse` w/o specifying `norm=False`.  Is this an issue you've seen discussed by others anywhere?  If we can reproduce the issue reliably, it should be pretty trivial to expand biom's test suite to cover this edge case you've observed (PR which implemented `inplace` is [here](https://github.com/biocore/biom-format/pull/351/files)).  I have a TON to learn in this realm, FYI.  😃    I'll say more about documentation later.